<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataProfile</class>
 <widget class="QDialog" name="DataProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Data Profile</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>280</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_9">
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="btnSave">
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="btnCancel">
      <property name="text">
       <string>Cancel</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="gbRandomValue">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>451</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Middle Text (Random Value)</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>31</y>
      <width>431</width>
      <height>56</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>min</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="sbRandomFrom">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbRandomTo">
       <property name="minimum">
        <number>-1000</number>
       </property>
       <property name="maximum">
        <number>1000</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbEndText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>End Text</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="txtEndText">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="gbStartText">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Start Text</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="txtStartText">
    <property name="geometry">
     <rect>
      <x>13</x>
      <y>31</y>
      <width>431</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>451</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Data Profile Name</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="txtProfileName"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
